new.php

```diff
@@ -113,6 +115,20 @@
 
 $wikiName = "Patch Demo ($patchesAppliedText)";
 
+$mainPage = "This wiki was automatically generated on [$server$serverPath Patch Demo] at ~~~~~ and applies the following patches:\n";
```

Comment:
MediaWiki has magic words for this, you can just write it like below instead of parsing the URLs:
```
[//{{SERVERNAME}} Patch Demo]
```
Or even this, but it seems to always return a HTTP URL on the existing wikis, possibly we should be setting `$wg`-something to override that:
```
[{{SERVER}} Patch Demo]
```

Reply:
Ah, I guess we need them anyway…

Reply:
That gets you to the root, but not if you have a path, e.g. myserver.com/patchdemo/ which currently works everywhere else.